README.md

```diff
@@ -102,3 +102,52 @@ const InstanceLogger = init(({ name }) => {
 
 const george = InstanceLogger({ name: 'George' }); // Created new instance: George
 ```
+
+
+### overrides()
```

Comment:
Not sure which blog post you're referencing. But the `overrides` is genuinely the name you came up with here: https://github.com/stampit-org/stamp-utils/issues/1#issuecomment-145359335

I'm open to suggestions because it feels like `overrides` is to generic name.